### ISSUE TITLE
Beautify the CLI Menu 

### ISSUE THREAD

**Issue by CS50X-RGB** [CS50X-RGB/File-Uploader#2]
Update the CLI Menu using nodejs tty package

**Comment by Mannxxx** [comment]
@CS50X-RGB , Do you mean node-ttl package?

**Comment by CS50X-RGB** [comment]
@Mannxxx sorry there was a typo its tty <a href="https://nodejs.org/api/tty.html">Click here</a> to see

**Comment by Mannxxx** [comment]
Yeah I'm working on this task. Could you please let me know what all commands do you need? 

So far, I've created basic layout of the CLI Menu:

![image](https://github.com/CS50X-RGB/CLI-Bluetooth./assets/107349957/1847c4f9-df9c-4479-896a-a2b5e3cf10f2)

**Comment by CS50X-RGB** [comment]
Send File after clicking it ask their name and store it and lastly the exit button that's it 
So 
Send File -> Ask Name (Store the user name)
And lastly the exit button

**Comment by CS50X-RGB** [comment]
> Send File after clicking it ask their name and store it and lastly the exit button that's it So Send File -> Ask Name (Store the user name) And lastly the exit button

And plz first get the issue assigned to you then start working on it

**Comment by Mannxxx** [comment]
@CS50X-RGB, Does this look fine?

![image](https://github.com/CS50X-RGB/CLI-Bluetooth./assets/107349957/b63bcbfc-8d1d-47c4-9cf1-1eb48067022e)

Are there any changes that should be made before I create the pull request?

**Comment by CS50X-RGB** [comment]
Yeah it's looking good nice work you can push your changes @Mannxxx
